### PR TITLE
Add type of return value for eventmit.

### PR DESCRIPTION
**Pull request by YayoiAyame**
I wanted to use the interface of return value for eventmit, so I added a type named `Mitter` to match the name.

### CODE REVIEW

File: src/eventmit.ts [hunk 1]
```diff
@@ -1,28 +1,36 @@
 export type EventmitHandler<T> = (value: T) => any;
-export const eventmit = <T>() => {
+
+export type Mitter<T> = {
```
Comment (azu) [inline src/eventmit.ts:3]:
```suggestion
export type Eventmitter = {
```

**Review by azu — COMMENTED** [review]
Thanks for PR.

`Eventmitter` is reasonable name?

File: src/eventmit.ts (hunk 1 above)
Comment (azu) [reply src/eventmit.ts:3]: We also need to update README.md
https://github.com/azu/eventmit#api (just copy paste)

Comment (YayoiAyame) [reply src/eventmit.ts]: OK, I fixed it.

**Comment by azu** [comment]
Thanks!

I've released [v1.1.0](https://github.com/azu/eventmit/releases/tag/v1.1.0)

**Pull request merged**